### DISCUSSION TITLE
The overlay when changing mania scroll speed doesn't show the currently-bound shortcut

### DISCUSSION THREAD

**Discussion by fyyrraaa** [ppy/osu#15210] [Q&A] [answered]
When I change the scroll speed ingame in mania, it seems like the shortcut is not shown in the overlay despite it being bound by default already.
![osu_2021-10-20_13-44-50](https://user-images.githubusercontent.com/3055647/138161945-7e8127be-95e6-4695-a5bc-c9a25103225f.png)

Here it shows that the shortcut is bound:
![osu_2021-10-20_13-48-37](https://user-images.githubusercontent.com/3055647/138162035-b6bb54d4-ff9b-4dd1-be0d-ef147c0ac8af.png)

Is this normal, or is it an issue on my side?

OS: macOS Big Sur 11.6
osu!lazer version: 2021.1016.0

**Comment by bdach** [comment, marked as answer]
lack of implementation, see https://github.com/ppy/osu/issues/15211
